galleries/examples/text_labels_and_annotations/rainbow_text.py

```diff
@@ -1,86 +1,32 @@
-"""
-============
-Rainbow text
-============
```

Comment:
The automatic check for the doc build is failing because it expects to find a docstring here.  I suggest to just reinstate this as far as the title if you don’t want to think about the details of it for now.  Then we will be able to see how your new version looks with its plot.